pool/db.go

```diff
@@ -368,10 +368,10 @@ func persistLastPaymentInfo(db *bolt.DB, height uint32, paidOn int64) error {
 	})
 }
 
-func loadLastPaymentInfo(db *bolt.DB) (uint32, uint64, error) {
+func loadLastPaymentInfo(db *bolt.DB) (uint32, int64, error) {
 	funcName := "loadLastPaymentInfo"
 	var height uint32
-	var paidOn uint64
+	var paidOn int64
```

Comment:
What's the reason for changing the type here and for `loadLastPaymentCreatedOn`? We are expecting any negative values for the time so I'd think maintaining `uint64` would adequate. Thoughts?

Reply:
I noticed the created-on/paid-on times were being cast to int64 so maintaining this avoids the casting. Lets maintain the type changes made here.